nvflare/edge/device/android/sdk/ETTrainerExecutor.kt

```diff
@@ -0,0 +1,93 @@
+package com.nvidia.nvflare.sdk
+
+import android.util.Log
+import com.nvidia.nvflare.sdk.core.DXO
+import com.nvidia.nvflare.sdk.core.Context
+import com.nvidia.nvflare.sdk.core.ContextKey
+import com.nvidia.nvflare.sdk.core.Signal
+import com.nvidia.nvflare.sdk.core.Executor
+import com.nvidia.nvflare.sdk.training.ETTrainer
+import com.nvidia.nvflare.sdk.training.TrainingConfig
+import kotlinx.coroutines.runBlocking
+
+/**
+ * ExecuTorch-specific implementation of Executor.
+ * Handles ET training execution with dataset creation and result conversion.
+ */
+class ETTrainerExecutor(
+    private val trainer: ETTrainer
+) : Executor {
+    private val TAG = "ETTrainerExecutor"
+
+    override fun execute(taskData: DXO, ctx: Context, abortSignal: Signal): DXO {
+        if (abortSignal.isTriggered) {
+            throw RuntimeException("Execution aborted")
+        }
+
+        try {
+            Log.d(TAG, "Starting training execution")
+            Log.d(TAG, "Task data keys: ${taskData.data.keys}")
+            Log.d(TAG, "Task data: $taskData")
+            
+            // Extract training configuration from task data
+            val trainingConfig = extractTrainingConfig(taskData, ctx)
+            
+            // Extract model data from task data
+            val modelData = taskData.data["model"] as? String
+            Log.d(TAG, "Extracted model data: ${modelData?.take(50) ?: "null"}...")
+            Log.d(TAG, "Model data length: ${modelData?.length ?: 0}")
+            Log.d(TAG, "Model data starts with '{': ${modelData?.startsWith("{")}")
+            Log.d(TAG, "Model data ends with '}': ${modelData?.endsWith("}")}")
+            Log.d(TAG, "Model data first 100 chars: ${modelData?.take(100)}")
+            Log.d(TAG, "Model data last 50 chars: ${modelData?.takeLast(50)}")
```

Comment:
some of debug logging let's remove in the next PR where you checkin the app code